components/accelerator/nvidia/peermem/component.go

```diff
@@ -126,10 +127,24 @@ func (c *component) Events(ctx context.Context, since time.Time) ([]components.E
 		if logItem.Matched == nil {
 			continue
 		}
+
 		if logItem.Matched.Name != dmesg.EventNvidiaPeermemInvalidContext {
 			continue
 		}
 
+		// skip this for now as the latest driver https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-560-35-03/index.html#abstract fixes this issue
+		// "4272659 – A design defect has been identified and mitigated in the GPU kernel-mode driver, related to the GPUDirect RDMA support
+		// in MLNX_OFED and some Ubuntu kernels, commonly referred to as the PeerDirect technology, i.e. the one using the peer-memory kernel
+		// patch. In specific scenarios, for example involving the cleanup after killing of a multi-process application, this issue may lead to
+		// use-after-free and potentially to kernel memory corruption."
+		//
+		// ref. https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-129-03/index.html
+		// ref. https://github.com/Mellanox/nv_peer_memory/issues/120
+		line := logItem.Line
```

Comment:
just copy line 130 and change the != to == done. remove all complicated regex stuff.

Reply:
it is ok to repeat that line and the ONLY difference is the operator ( == )